python/mxnet/libinfo.py

```diff
@@ -76,6 +76,33 @@ def find_lib_path():
         os.environ['PATH'] = os.environ['PATH'] + ';' + os.path.dirname(lib_path[0])
     return lib_path
 
+def find_include_path():
+    """Find MXNet dynamic library files.
+     Returns
+    -------
+    incl_path : string
+        Path to the header files.
+    """
+    incl_from_env = os.environ.get('MXNET_INCLUDE_PATH')
+    if incl_from_env:
+        if os.path.isfile(incl_from_env):
+            if not os.path.isabs(incl_from_env):
+                logging.warning("MXNET_INCLUDE_PATH should be an absolute path, instead of: %s",
+                                incl_from_env)
+            else:
+                if os.name == 'nt':
+                    os.environ['PATH'] = os.environ['PATH'] + ';' + os.path.dirname(incl_from_env)
+                return [incl_from_env]
+        else:
+            logging.warning("MXNET_INCLUDE_PATH '%s' doesn't exist", incl_from_env)
+    curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
+    incl_path = os.path.join(curr_path, '../../include/')
```

Comment:
'../../include/' looks hacky. If this file directory changes, this may fail.

Reply:
We discussed offline and found this follows the same pattern in find_lib_path()